oshi-core/src/main/java/oshi/software/os/windows/WindowsFileSystem.java

```diff
@@ -181,7 +181,7 @@ public WindowsFileSystem() {
         aVolume = new char[BUFSIZE];
 
         hVol = Kernel32.INSTANCE.FindFirstVolume(aVolume, BUFSIZE);
-        if (hVol == WinBase.INVALID_HANDLE_VALUE) {
+        if (hVol.equals(WinBase.INVALID_HANDLE_VALUE)) {
```

Comment:
again better to cal the `equals()` on `INVALID_HANDLE_VALUE`

Reply:
Thanks again. Switched it.